packages/editor/src/client/panel/userEditPanel.tsx

```diff
@@ -207,6 +210,23 @@ export function UserEditPanel({id, onClose, onSave}: UserEditPanelProps) {
                 <Button appearance="primary" onClick={() => setIsResetUserPasswordOpen(true)}>
                   {t('userList.panels.resetPassword')}
                 </Button>
+                <Button
+                  appearance="primary"
+                  style={{marginLeft: '20px'}}
+                  disabled={!email || !active}
+                  onClick={async () => {
+                    try {
+                      await sendWebsiteLogin({variables: {email}})
+                      Alert.success(
+                        t('userList.panels.sendWebsiteLoginSuccessMessage', {email}),
+                        2000
+                      )
+                    } catch (error) {
+                      Alert.error(t('userList.panel.sendWebsiteLoginFailureMessage', {error}), 0)
+                    }
+                  }}>
+                  {t('userList.panels.sendWebsiteLogin')}
+                </Button>
```

Comment:
May be line 218 to 226 can be separated into a method e.g handleSendWebsiteLogin